mitodl/mit-open
# score cutoff

### What are the relevant tickets?
closes https://github.com/mitodl/hq/issues/4985

### Description (What does it do?)
This PR adds a slider to the admin section of the search controls to specify the minimum score needed for results to be shown

### Screenshots (if appropriate):
<img width="1724" alt="Screenshot 2024-08-26 at 3 20 04 PM" src="https://github.com/user-attachments/assets/35aeb2bd-d014-45db-92a6-27e4b8057723">
<img width="332" alt="Screenshot 2024-08-26 at 3 20 32 PM" src="https://github.com/user-attachments/assets/d7ae6c72-2d51-405e-b22b-a8674fec9e4a">


### How can this be tested?
Log in as an admin user. Go to the search page. Make a text search - verify that as you increase the Minimum Score Cutoff with the slider, the number of results goes down but the order of the top results stays the same. The score cutoff should work with all three search modes
